specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessmentMetadata.json

```diff
@@ -495,16 +496,42 @@
               {
                 "value": "CustomerManaged",
                 "description": "User assessments pushed directly by the user or other third party to Azure Security Center"
+              },
+              {
+                "value": "VerifiedPartner",
+                "description": "An assessment that was created by a verified 3rd party if the user connected it to ASC"
               }
             ]
           }
+        },
+        "partnerData": {
+          "$ref": "#/definitions/SecurityAssessmentMetadataPartnerData"
```

Comment:
is this popoulated only when enum type is VerifiedPartner ?
Ideally, discriminator pattern is recommended , instead of on/off fields..

Reply:
it is only populated when the enum, is VerifiedPartner but we prefer not to use the discriminator pattern here since it usually make the API a little harder to use (customer feedback that we have)
so we would like to only put null here in case this is not a VerfiedPartner
anyway, we validate that the user do not populate this field unless needed and return an indicative error message if he did